src/features/copy-paste-menu-selection-override.js

```diff
@@ -0,0 +1,215 @@
+import ContentFeature from "../content-feature.js";
+
+export default class CopyPasteMenuSelectionOverride extends ContentFeature {
+    init () {
+        try {
+            (() => {
+                var lastContextMenuEventX = -1;
+                var lastContextMenuEventY = -1;
+                var lastContextMenuEventWasPrevented = false;
+                var isSelectionColorOverrideStyleAdded = false;
+
+                // - Suppress alerts displayed on right click
+                document.addEventListener('DOMContentLoaded', function () {
+                    // add alert suppression script to page content world.
+                    // use invisible `_ddg-suppress-alert-flag` element to pass `shouldSuppressAlert` value
+                    // from client content world script handling `contextmenu` events.
+                    var alertSuppressionScript = document.createElement('script');
+                    alertSuppressionScript.textContent = `
+                    (() => {
```

Comment:
I don't think this needs to be an injected script and instead just a method.

Reply:
the reason behind this is the `ContentScopeScript` being injected as a `defaultClient` scope script and the `alert` method is called by the document from the `page` scope, so they don‘t intersect, this hack is here to pass `shouldSuppressAlert` flag between the `page` and `defaultClient` scope and handle the `alert` call in the `page` scope. I‘m open for suggestions on how to handle this in "the right way".